### PR TITLE
Use aria descriptions instead of labels for TextWithTooltip

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/25398

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Use aria descriptions instead of labels for TextWithTooltip ([\#10952](https://github.com/matrix-org/matrix-react-sdk/pull/10952)). Fixes vector-im/element-web#25398.

### CODE REVIEW

File: src/components/views/elements/TextWithTooltip.tsx [hunk 1]
```diff
@@ -28,19 +29,18 @@ interface IProps extends HTMLAttributes<HTMLSpanElement> {
 }
 
 export default class TextWithTooltip extends React.Component<IProps> {
+    private id = "mx_TextWithTooltip_" + randomString(8);
```
Comment (robintown) [inline src/components/views/elements/TextWithTooltip.tsx:32]: Is there a reason why TooltipTarget doesn't automatically generate an ID internally?

**Review by robintown — COMMENTED** [review]
Trying to understand why things are this way

File: src/components/views/elements/TextWithTooltip.tsx (hunk 1 above)
Comment (t3chguy) [reply src/components/views/elements/TextWithTooltip.tsx:32]: Good point

**Review by robintown — APPROVED** [review]
Thanks, it seems a lot harder to misuse now!

**Pull request merged**